pages/api/content/fr/glamorous-component-factory.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  title: '`glamorousComponentFactory`',
+  subtitle: '',
+  description: `
+    Si vous en créez un vous-même ou utilisez l'un des éléments intégrés mentionnés ci-dessus, chaque ~glamorousComponentFactory~ vous permet de l'appeler avec des styles et vous renvoie un nouveau composant qui aura ces styles appliqués lors de son rendu. Ceci est réalisé par la génération d'un ~className~ pour les styles donnés et par le transfert du ~className~ sur l'élément rendu. Donc, si vous envelopper un composant que vous avez l'intention de styliser, vous devez vous assurer que vous acceptez le ~className~ comme une prop et que vous l'appliquer à l'endroit où vous voulez que les styles soient appliqués dans votre composant personnalisé (normalement l'élément racine).
```

Comment:
@kentcdodds
As for the line lengths, I can do it, in fact I respect the line breaks of the original version. This makes it easier to manage updates from the original.

I think we should start with the original. I would adapt after the French version.